packages/@o3r-training/training-tools/README.md

```diff
@@ -34,6 +34,19 @@ const serializedFiles = await getFilesTree([{
   }], instance.fs as FileSystem, EXCLUDED_FILES_OR_DIRECTORY);
 ```
 
+## Options
```

Comment:
is this supposed to be here? looks like a copy paste of `@o3r/artifactory-tools`

Reply:
indeed I forgot to include the modifications, should be updated now